api/proto/teleport/accesslist/v1/accesslist_service.proto

```diff
@@ -112,6 +112,10 @@ service AccessListService {
 
   // GetInheritedGrants returns the inherited grants for an access list.
   rpc GetInheritedGrants(GetInheritedGrantsRequest) returns (GetInheritedGrantsResponse);
+
+  // ListUserAccessLists returns a paginated list of all access lists where the
+  // user is explicitly an owner or member
```

Comment:
You mentioned in a comment that this differs from the other `ListAccessListV2` rpc in that this one "returns the users' owner / member assignments under `CurrentUserAssignments`". Can you add that to the description here, perhaps with an expanded explanation of what `CurrentUserAssignments` means?